uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -0,0 +1,579 @@
+/*******************************************************************************
+ *  Copyright 2012-2022 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+import Esri.ArcGISRuntime.Toolkit.Controller 100.14
+import QtQuick 2.12
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.15
+import QtGraphicalEffects 1.12
+
+/*!
+  \qmltype FloorFilter
+  \inqmlmodule Esri.ArcGISRuntime.Toolkit
+  \ingroup ArcGISQtToolkitUiQmlViews
+  \since 100.14
+  \brief Allows to display and filter the available floor aware layers in the current \c GeoModel.
+
+  The FloorFilter allows the interaction with the available floor aware layers. A user can select from a list of sites which presents
+  their facilities. Once a facility is chosen, it is possible to toggle between its levels which will show them on the \c GeoView.
+  2D maps and 3D scenes are supported.
```

Comment:
I think this brief could use some updates to give more of an overview of the functionality and to avoid passive voice. Here is a suggestion

```suggestion
  This component supports manual selection and text-based filtering for sites and facilities.
  Once a site and facility is selected, levels can be selected and the GeoView will be updated.
  
  In addition to the browsing workflow where sites and facilities are manually selected, automatic facility 
  selection based on the GeoView's current viewpoint. This mode is useful when you are exploring the
  GeoView but do not know the name of the site or facility.
  
  The user interface is driven from the FloorAware data that is available in the GeoModel's FloorManager. 
  
  2D maps and 3D scenes are supported.
```

Reply:
I would also to enphasise the existence of the `allSite` button. Something like this:
...
This mode is useful when you are exploring the GeoView but do not know the name of the site or facility.
`For the same reason, it is possible to list all the facilties available in the \c FloorManager using the button on the siteView.
`

Reply:
Yes, Good Idea! 👍